fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalJoin.java

```diff
@@ -132,7 +165,10 @@ public List<Slot> computeOutput(Plan leftInput, Plan rightInput) {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("LogicalJoin (").append(joinType);
-        condition.ifPresent(expression -> sb.append(", ").append(expression));
+        sb.append(" [");
+        hashJoinConjuncts.stream().map(expr -> sb.append(" ").append(expr)).collect(Collectors.toList());
```

Comment:
please refer to PhysicalJoin's toString's comment

Reply:
done